AuthenticationAndAuthorization/BusinessLogic/Model/RegisterUserRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusinessLogic
+{
+    public record RegisterUserRequest
+    (
+        [Required] string Username,
+        [Required] string Email,
+        [Required] string PasswordHash
```

Comment:
You cannot send password hash. Instead you should send password to the server, then your UserManager can register user with some method which is something like `RegisterUser(user, password)`.